prebid-mobile/adops-line-item-setup-adform.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: page
+title: Setup Line Items for Adform
+description: Setup line items for Adform
+pid: 2
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile-adops
+---
+
+
+<div class="bs-docs-section" markdown="1">
+
+# Step by Step Line Item Setup for Adform
+
+* TOC
+{:toc }
+
+This page describes step by step how to set up Prebid Mobile line items for Adform to serve ads on app with the Prebid SDK.
+
+
+## Step 1. Create a Campaign
+
+- Create a client under which you would like to run in-app HB campaign (1)
+- Select the longest possible period (2).
+- Set goal to **Unlimited** (3).
+
+{: .pb-med-img :}
+  ![Adform Campaign Setup]({{site.github.url}}/assets/images/prebid-mobile/adops-line-item-setup-adform/adform1.png "Example Adform Campaign Setup")
+
+## Step 2. Create a Line Item
+
+In the General settings Details section:
+- Select delivery type: **Non-guaranteed** (1)
+- Select status: **Booked** (2)
+
+{: .pb-med-img :}
+  ![Adform Line Item Setup]({{site.github.url}}/assets/images/prebid-mobile/adops-line-item-setup-adform/adform2.png "Example Adform Line Item")
+
+
+- Select full period available according to the campaign setup.
+
+{: .pb-med-img :}
+  ![Adform Line Item Setup]({{site.github.url}}/assets/images/prebid-mobile/adops-line-item-setup-adform/adform3.png "Example Adform Line Item")
+
+In the General settings Pricing & Budget section:
+- Select **dCPM** pricing model. It allows taking bid information dynamically each time a request is sent to the ad server together with the winning bid information (4).
+- Default price should stay set at zero (5).
+- Currency should be the same as the one used in the prebid initiated web auction. Selecting wrong currency could result in discrepancies in the report (6).
+- **Net/Gross** value should be selected according to the individual SSP recommendations (7).
+
+{: .pb-med-img :}
+  ![Adform Line Item Setup]({{site.github.url}}/assets/images/prebid-mobile/adops-line-item-setup-adform/adform4.png "Example Adform Line Item")
+
+In the Inventory section:
+Select placements that are implemented on in-app and should run via Prebid Mobile SDK.
+
+In the Targeting section:
+In the key-value management section create a key-value pair that will represent each bidder.
+For example, key: `hb_bidder`, value: `bidderName1` or `hb_bidder`, value: `bidderName2`
+
+In oder to create a key-value pair, follow these steps:
+1. Press **Create Category** (1) and insert name: `hb_bidder` (8)
+2. Go into the key folder and create separate values for it: `bidderName2`, `bidderName1` (9).
```

Comment:
Are these case-sensitive? Value casing in the text doesn't match casing in the screen shot.

Reply:
It is case-sensitive. I've added explanation and changed values as in screenshot.